eng/common/scripts/Verify-Links.ps1

```diff
@@ -117,13 +119,26 @@ function CheckLink ([System.Uri]$linkUri)
   if ($checkedLinks.ContainsKey($linkUri)) { return }
 
   Write-Verbose "Checking link $linkUri..."
+
+  # Check if link uri includes locale info.
+  if ($linkGuidance) {
```

Comment:
Lets move this to the bottom of this function and we should make sure we don't add this to badLinks more then once

Reply:
Will move to bottom. If we have a broken link with locale, then we will print warning messages, but only display one bad link.